### ISSUE TITLE
LWP3 repl initial traceback

### ISSUE THREAD

**Issue by vascolp** [pybricks/pybricksdev#91]
Hi!

When I open `pybricksdev lwp3 repl`, after connecting to a device, it gets a huge traceback (see below).
Then it seems to work fine...
Below:
* traceback
* installed packages (pip list)

Environment: Windows 11, Python 3.12.3, pybricksdev 1.0.0a49.

Traceback:
```
pybricksdev lwp3 repl
[08:43:35.871] INFO: scanning...
[08:43:42.653] INFO: found device
[08:43:44.894] INFO: connected
--- Logging error ---
Traceback (most recent call last):
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 1160, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 999, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 703, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 392, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 537, in __repr__
    return f"{self.__class__.__name__}({repr(self.port)}, {repr(self.device)}, {repr(self.hw_ver)}, {repr(self.fw_ver)})"
                                             ^^^^^^^^^
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 494, in port
    return PortID(self._data[3])
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\enum.py", line 761, in __call__
    raise TypeError(
TypeError: <enum 'PortID'> has no members; specify `names=()` if you meant to create a new, empty, enum
Call stack:
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\Program Files\Python312\Scripts\pybricksdev.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\__init__.py", line 389, in main
    asyncio.run(subparsers.choices[args.tool].tool.run(args))
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 674, in run_until_complete
    self.run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\windows_events.py", line 322, in run_forever
    super().run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 641, in run_forever
    self._run_once()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 1987, in _run_once
    handle._run()
  File "D:\Program Files\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\lwp3\repl.py", line 159, in handle_notify
    logger.info("received: %s", msg)
Unable to print the message and arguments - possible formatting error.
Use the traceback above to help find the error.
--- Logging error ---
Traceback (most recent call last):
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 1160, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 999, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 703, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 392, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 537, in __repr__
    return f"{self.__class__.__name__}({repr(self.port)}, {repr(self.device)}, {repr(self.hw_ver)}, {repr(self.fw_ver)})"
                                             ^^^^^^^^^
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 494, in port
    return PortID(self._data[3])
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\enum.py", line 761, in __call__
    raise TypeError(
TypeError: <enum 'PortID'> has no members; specify `names=()` if you meant to create a new, empty, enum
Call stack:
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\Program Files\Python312\Scripts\pybricksdev.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\__init__.py", line 389, in main
    asyncio.run(subparsers.choices[args.tool].tool.run(args))
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 674, in run_until_complete
    self.run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\windows_events.py", line 322, in run_forever
    super().run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 641, in run_forever
    self._run_once()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 1987, in _run_once
    handle._run()
  File "D:\Program Files\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\lwp3\repl.py", line 159, in handle_notify
    logger.info("received: %s", msg)
Unable to print the message and arguments - possible formatting error.
Use the traceback above to help find the error.
--- Logging error ---
Traceback (most recent call last):
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 1160, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 999, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 703, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\logging\__init__.py", line 392, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 537, in __repr__
    return f"{self.__class__.__name__}({repr(self.port)}, {repr(self.device)}, {repr(self.hw_ver)}, {repr(self.fw_ver)})"
                                             ^^^^^^^^^
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\ble\lwp3\messages.py", line 494, in port
    return PortID(self._data[3])
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\enum.py", line 761, in __call__
    raise TypeError(
TypeError: <enum 'PortID'> has no members; specify `names=()` if you meant to create a new, empty, enum
Call stack:
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\Program Files\Python312\Scripts\pybricksdev.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\__init__.py", line 389, in main
    asyncio.run(subparsers.choices[args.tool].tool.run(args))
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
  File "D:\Program Files\Python312\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 674, in run_until_complete
    self.run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\windows_events.py", line 322, in run_forever
    super().run_forever()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 641, in run_forever
    self._run_once()
  File "D:\Program Files\Python312\Lib\asyncio\base_events.py", line 1987, in _run_once
    handle._run()
  File "D:\Program Files\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "D:\Program Files\Python312\Lib\site-packages\pybricksdev\cli\lwp3\repl.py", line 159, in handle_notify
    logger.info("received: %s", msg)
Unable to print the message and arguments - possible formatting error.
Use the traceback above to help find the error.
Type message and press ENTER to send. Press CTRL+D to exit.
>>> HubPropertyRequestUpdate(HubProperty.NAME)

[08:43:50.543] INFO: sending: HubPropertyRequestUpdate(<HubProperty.NAME: 1>)
[08:43:50.685] INFO: received: HubPropertyUpdate(<HubProperty.NAME: 1>, 'abc')
>>>
```

Installed packages
```
pip list
Package                                                 Version
------------------------------------------------------- --------
aioserial                                               1.3.1
appdirs                                                 1.4.4
argcomplete                                             1.12.3
asyncssh                                                2.14.2
bleak                                                   0.22.2
cffi                                                    1.16.0
colorama                                                0.4.6
cryptography                                            42.0.8
distlib                                                 0.3.8
filelock                                                3.14.0
hidapi                                                  0.14.0
mpy-cross-v5                                            1.0.2
mpy-cross-v6                                            1.0.2
packaging                                               22.0
pip                                                     24.0
platformdirs                                            4.2.1
prompt_toolkit                                          3.0.47
pybricksdev                                             1.0.0a49
pycparser                                               2.22
pyserial                                                3.5
pyusb                                                   1.2.1
reactivex                                               4.0.4
semver                                                  2.13.0
setuptools                                              70.1.1
tqdm                                                    4.66.4
typing_extensions                                       4.12.2
virtualenv                                              20.26.1
wcwidth                                                 0.2.13
winrt-runtime                                           2.1.0
winrt-Windows.Devices.Bluetooth                         2.1.0
winrt-Windows.Devices.Bluetooth.Advertisement           2.1.0
winrt-Windows.Devices.Bluetooth.GenericAttributeProfile 2.1.0
winrt-Windows.Devices.Enumeration                       2.1.0
winrt-Windows.Foundation                                2.1.0
winrt-Windows.Foundation.Collections                    2.1.0
winrt-Windows.Storage.Streams                           2.1.0
```

**Comment by dlech** [comment]
Which device and firmware version are you connecting to?

**Comment by vascolp** [comment]
Its a City Hub with LEGO Firmware just flashed via Control-+ App... Yesterday.

**Comment by dlech** [comment]
Fix released in v1.0.0-alpha.50

**Comment by vascolp** [comment]
Confirmed!
```
 pybricksdev lwp3 repl
[18:23:08.635] INFO: scanning...
[18:23:09.283] INFO: found device
[18:23:11.469] INFO: connected
[18:23:11.526] INFO: received: HubIOAttachedMessage(<PortID.50: 50>, <IODeviceKind.HUB_STATUS_LIGHT: 23>, Version(0x10000000), Version(0x10000000))
[18:23:11.540] INFO: received: HubIOAttachedMessage(<PortID.59: 59>, <IODeviceKind.HUB_BATTERY_CURRENT: 21>, Version(0x00000002), Version(0x00000002))
[18:23:11.541] INFO: received: HubIOAttachedMessage(<PortID.60: 60>, <IODeviceKind.HUB_BATTERY_VOLTAGE: 20>, Version(0x00000002), Version(0x00000002))
Type message and press ENTER to send. Press CTRL+D to exit.
>>> HubPropertyRequestUpdate(HubProperty.FW_VERSION)

[18:23:27.922] INFO: sending: HubPropertyRequestUpdate(<HubProperty.FW_VERSION: 3>)
[18:23:28.089] INFO: received: HubPropertyUpdate(<HubProperty.FW_VERSION: 3>, Version(0x11020000))
>>> HubPropertyRequestUpdate(HubProperty.BATTERY_VOLTAGE)

[18:23:37.545] INFO: sending: HubPropertyRequestUpdate(<HubProperty.BATTERY_VOLTAGE: 6>)
[18:23:37.689] INFO: received: HubPropertyUpdate(<HubProperty.BATTERY_VOLTAGE: 6>, 0)
>>>

```
Thanks!!!
